### PR TITLE
Update dependency on elastic-package

### PR DESCRIPTION
**Pull request by mtojek**
This PR bumps up dependency on the elastic-package (changes around semantic validation of spec).

### CODE REVIEW

**Comment by mtojek** [comment]
@ycombinator I can't figure out which format of security_rule is correct:

In spec:
https://github.com/elastic/package-spec/blob/master/test/packages/good/kibana/security_rule/000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json

In package:
https://github.com/elastic/integrations/blob/master/packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json

Please look at the "rule_id" and "id".

**Comment by rw-access** [comment]
Oh! I didn't realize spec was validating anything other than JSON so the spec was out of date. The format inside the integration that's structured like a SO is correct.

**Comment by mtojek** [comment]
Thanks @rw-access for clarifying this. It seems that we need to fix the package-spec.

File: packages/mongodb/kibana/visualization/mongodb-operation-counters.json [hunk 1]
```diff
@@ -175,7 +175,7 @@
     ],
     "references": [
         {
-            "id": "mongodb-MongoDB-search",
```
Comment (ycombinator) [inline packages/mongodb/kibana/visualization/mongodb-operation-counters.json:178]: These changes seem unrelated to the security rule naming changes?

File: packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
             "https://developer.okta.com/docs/reference/api/event-types/"
         ],
         "risk_score": 21,
-        "rule_id": "000047bb-b27a-47ec-8b62-ef1a5d2c9e19",
+        "rule_id": "rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19",
```
Comment (rw-access) [inline packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json:25]: wait this isn't right. I don't understand why this happened

File: packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json [hunk 3]
```diff
@@ -36,6 +36,6 @@
         "type": "query",
         "version": 5
     },
-    "id": "000047bb-b27a-47ec-8b62-ef1a5d2c9e19",
+    "id": "rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19",
```
Comment (rw-access) [inline packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json:39]: Or this change

**Review by rw-access — CHANGES_REQUESTED** [review]

File: packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json (hunk 2 above)
Comment (mtojek) [reply packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json:25]: We have a rule that ID should consistent with filename.

File: packages/mongodb/kibana/visualization/mongodb-operation-counters.json (hunk 1 above)
Comment (mtojek) [reply packages/mongodb/kibana/visualization/mongodb-operation-counters.json:178]: Linting failed. It seems that the spec hasn't been updated.

Comment (mtojek) [reply packages/mongodb/kibana/visualization/mongodb-operation-counters.json:178]: Ah, I see. You're referring to the commit name.

File: packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json (hunk 2 above)
Comment (rw-access) [reply packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json:25]: The rule ID can't change, it's defined elsewhere. So we could change the filename. That's fine with me.

I don't really understand why it's a hard and fast requirement or why the change is necessary, to be honest.

Comment (mtojek) [reply packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json:25]: It's for readability and clarity purposes. We encountered misnaming in the past, e.g. "access-logs-dashboard.json" which contains "error-logs-overview". This way we simplify navigating through resources.

File: packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json (hunk 3 above)
Comment (mtojek) [reply packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json]: Fixed

File: packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json (hunk 2 above)
Comment (mtojek) [reply packages/security_detection_engine/kibana/security_rule/rule-000047bb-b27a-47ec-8b62-ef1a5d2c9e19.json]: Fixed

**Review by rw-access — APPROVED** [review]
LGTM. thanks for your patience @mtojek

**Review by ycombinator — APPROVED** [review]
LGTM.
